common/primitives/src/schema.rs

```diff
@@ -1,5 +1,19 @@
-use frame_support::BoundedVec;
+#[cfg(feature = "std")]
+use crate::utils;
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_std::prelude::*;
+#[cfg(feature = "std")]
+use utils::*;
 
 pub type SchemaId = u16;
 
-pub type Schema<T> = BoundedVec<u8, T>;
```

Comment:
we might need the `Schema` in here later but we can move it when we needed it in the worst case. I think `SchemaResponse` will reduce the requirement to access `Schema` directly, but let's see.

Reply:
lol @saraswatpuneet asked me to move it to there